frontend/app/components/wp-table/wp-table.directive.html

```diff
@@ -131,7 +131,8 @@
             </accessible-checkbox>
             <span>
               <a class="wp-table--details-link"
-                 ui-sref="work-packages.list.details.overview({workPackageId: row.object.id})">
+                 ui-state="desiredSplitViewState"
+                 ui-state-params="{workPackageId: row.object.id}">
```

Comment:
can a one time binding be used for the params?

/cc @romanroe any thought about performance regarding this approach?